circleci/catapult-publish

```diff
@@ -76,6 +76,7 @@ if [ -f go.mod ]; then
     post_github_pr_comment $USER $REPO_NAME $FULL_SHA $GH_RELEASE_TOKEN "$VALIDATE_OUTPUT"
   fi
 
+  set -e
```

Comment:
`set -e` is already at the begging of the file, is there a reason why it's needed later on?

Reply:
At the start of the validation section, we disable this flag.